### PR TITLE
Update UserTest.php

### PR DESCRIPTION
**Pull request by ducit95**

### CODE REVIEW

File: tests/UserTest.php [hunk 1]
```diff
@@ -20,7 +20,6 @@ public function testExample()
      public function testGedata()
```
Comment (kakeyang) [inline tests/UserTest.php]: You check only status code but can it be test?
I think these test cases can pass with invalid input but...

File: tests/UserTest.php [hunk 2]
```diff
@@ -20,7 +20,6 @@ public function testExample()
...
     {
```
Comment (kakeyang) [inline tests/UserTest.php:26]: You'd better add more test cases of error / boundary conditions and security.

Comment (ducit95) [inline tests/UserTest.php:26]: i use angularjs validate all form so in PHPUnit i don't test for validate
